opensquirrel/replacer.py

```diff
@@ -43,10 +43,9 @@ def visit_controlled_gate(self, controlled_gate: ControlledGate):
         return result
 
 
-def get_replacement_matrix(replacement: List[Gate], qubit_mappings):
+def get_replacement_matrix(replacement: List[Gate], qubit_mappings: list[Qubit]):
```

Comment:
`get_replacement_matrix` is a name that doesn't match anymore with the uses of this function
- maybe move it to `matrix_expander.py`, along with the Reindexer
- maybe rename to `get_matrix_after_qubit_remapping` or a better name